plugins/faustwp/includes/settings/callbacks.php

```diff
@@ -372,6 +374,12 @@ function display_enable_disable_fields() {
 			<input type="checkbox" id="enable_image_source" name="faustwp_settings[enable_image_source]" value="1" <?php checked( $enable_image_source ); ?> />
 			<?php esc_html_e( 'Use the WordPress domain for media URLs in post content', 'faustwp' ); ?>
 		</label>
+		<br />
+
+		<label for="enable_usage_tracking">
+			<input type="checkbox" id="enable_usage_tracking" name="faustwp_settings[enable_usage_tracking]" value="1" <?php checked( $enable_usage_tracking ); ?> />
+			<?php esc_html_e( 'Opt into anonymous usage tracking to help us make Faust.js better.', 'faustwp' ); ?>
```

Comment:
Should we update "Faust.js" to be "FaustWP" or simply "Faust"?

Reply:
We may want to consider dropping the period at the end of this label since the other checkbox options do not have one.